SimpleAL.podspec

```diff
@@ -1,10 +1,10 @@
 Pod::Spec.new do |s|
   s.name         = 'SimpleAL'
-  s.version      = '1.0.1'
+  s.version      = '2.0.0'
```

Comment:
Oh didn't expect this to require a major version bump, how does adding these annotations break things?